chapters/VK_NV_ray_tracing/raytracing-resources.txt

```diff
@@ -165,8 +165,6 @@ include::../../api/enums/VkBuildAccelerationStructureFlagBitsNV.txt[]
   * ename:VK_BUILD_ACCELERATION_STRUCTURE_ALLOW_UPDATE_BIT_NV indicates that
     the specified acceleration structure can: be updated with pname:update
     of ename:VK_TRUE in flink:vkCmdBuildAccelerationStructureNV.
-    This flag must: only be set on a build that is not an update or is an
-    update with a source that also had this flag set.
```

Comment:
Already the VU's say all of it incl this flag **must** be identical. This sentence IMO only confuses.